### PR TITLE
BG-14220: Add erc and terc

### PR DESCRIPTION
**Pull request by thomas4019**
Add template for adding erc20 tokens as ofc tokens.

### CODE REVIEW

File: src/ofc.ts [hunk 1]
```diff
@@ -21,9 +22,11 @@ export class OfcCoin extends BaseCoin {
...
 
   constructor(options: OfcConstructorOptions) {
-    super({
-      ...options,
-    });
+    const {
+      addressCoin,
+      ...baseOptions
+    } = options;
+    super(baseOptions);
```
Comment (tylerlevine) [inline src/ofc.ts]: Needs assignment

Comment (ShanniL) [reply src/ofc.ts]: super(options)?

Comment (thomas4019) [reply src/ofc.ts]: Fixed.

File: src/ofc.ts [hunk 2]
```diff
@@ -125,3 +132,90 @@ export function tofc(
...
+ * @param fullName Complete human-readable name of the coin
+ * @param network Network object for this coin
+ * @param decimalPlaces Number of decimal places this coin supports (divisibility exponent)
+ * @param asset Asset which this coin represents. This is the same for both mainnet and testnet variants of a coin.
+ * @param kind Differentiates coins which represent fiat assets from those which represent crypto assets
+ * @param prefix? Optional coin prefix. Defaults to empty string
+ * @param suffix? Optional coin suffix. Defaults to coin name.
+ * @param isToken? Whether or not this account coin is a token of another coin
+ * @param features? Features of this coin. Defaults to the DEFAULT_FEATURES defined in `OfcCoin`
+ */
```
Comment (ShanniL) [inline src/ofc.ts]: minor thing but here needs addressCoin 0.0

**Pull request merged**